.github/workflows/ci.yml

```diff
@@ -30,17 +31,15 @@ jobs:
         echo '::1         localhost localhost.localdomain localhost6 localhost6.localdomain6' >> %SystemRoot%\System32\drivers\etc\hosts
       shell: cmd
     - uses: actions/checkout@v4
-    - name: Set up JDK ${{ matrix.jdk-distribution }} 17
+    - name: Set up JDK ${{ matrix.jdk-distribution }} ${{ matrix.jdk-version }}
       uses: actions/setup-java@v4
       with:
         distribution: ${{ matrix.jdk-distribution }}
-        java-version: |
-          11
-          17
```

Comment:
To use the multi-JDK support provided by `jboss-parent`, add 21 to this list as the last entry.

Reply:
Done, please review